packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -26,6 +26,7 @@ import {
   filterClauses,
   FilterClause
 } from '../../../store/resource'
+import { truncate } from 'lodash'
```

Comment:
I don't think we need this function here. Also quick note, you won't wanna use lodash in your project coz there's most likely a better way. And ... if you were actually gonna use lodash, its better to import using `import truncate from 'lodash/truncate'`, that way we enjoy tree shaking from webpack, pulling in only required functions from the package.

But yeah, we don't need it in this case 😅